ocrd_froc/processor.py

```diff
@@ -37,50 +36,50 @@ def __init__(self, *args, **kwargs):
     def setup(self):
 
         if 'network' not in self.parameter:
-            self.parameter['network'] = resource_filename(__name__, 'models/default.froc')
-        
+            self.parameter['network'] = str(resource_filename(f'ocrd_froc.models', 'default.froc'))
+
         network_file = self.resolve_resource(self.parameter['network'])
         self.froc = Froc.load(network_file)
 
     def _process_segment(self, segment, image):
         textStyle = segment.get_TextStyle()
+        if textStyle and self.parameter['replace_textstyle']:
+            textStyle = None
```

Comment:
This effectively throws away information. We usually want to incrementally annotate, which each processor adding more details.

So while this processor is versed in fontFamily, another might be specialised on typestyle (e.g. serif, bold, italic, underlined, letterSpaced) or colour (textColour, reversedVideo).

Also, if the classifier below does not actually produce a result (because the confidence is too low, or whatever), then we might even throw away existing fontFamily without adding a new one.

So I would suggest keeping the TextStyle element, and only overwriting it **selectively** and **on demand** (if overwriting is enabled).